### ISSUE TITLE
Can't remove the plugin

### ISSUE THREAD

**Issue by DDayOverlord** [samiahmedsiddiqui/custom-permalinks#84]
I need to remove the plugin for a very important test but I meet a major website failure anytime I deactivate the plugin.

The links are incorrect: they point to other pages.

how can I remove the plugin without crashing my entire site?

Thanks 

Marc

**Comment by samiahmedsiddiqui** [comment]
Hi Marc, 

Sorry for the later reply. Are you still having issues with that?

**Comment by DDayOverlord** [comment]
Yes I do. Can you help?Thanks Marc Le 9 juil. 2024 à 11:38, Sami Ahmed Siddiqui ***@***.***> a écrit :﻿
Hi Marc,
Sorry for the later reply. Are you still having issues with that?
